src/DynamoSandbox/DynamoSandbox.csproj

```diff
@@ -15,9 +15,7 @@
     <RuntimeIdentifier>win-x64</RuntimeIdentifier>
   </PropertyGroup>
 
-	<Target Name="NpmRunBuild" BeforeTargets="BeforeBuild">
```

Comment:
?? whats up with this change, did you intend to move this to DynamoCoreWpf?

Reply:
I suggest simply revert all changes to this file because all of these will be cleaned up in my other PR, then you will have conflicts